qutip/bloch.py

```diff
@@ -38,16 +38,28 @@
 from numpy import (ndarray, array, linspace, pi, outer, cos, sin, ones, size,
                    sqrt, real, mod, append, ceil, arange)
 
+from packaging.version import parse as parse_version
+
 from qutip.qobj import Qobj
 from qutip.expect import expect
 from qutip.operators import sigmax, sigmay, sigmaz
 
 try:
+    import matplotlib
     import matplotlib.pyplot as plt
     from mpl_toolkits.mplot3d import Axes3D
     from matplotlib.patches import FancyArrowPatch
     from mpl_toolkits.mplot3d import proj3d
 
+    def matplotlib_version_gte(version='3.4'):
```

Comment:
I wouldn't have a default value here at all - it's not obvious why it's 3.4 (somebody two months in the future will have no idea), so it's not clear what `matplotlib_version_gte()` will do.  Better to make all the calls `matplotlib_version_gte('3.4')` or so.

(a minor style note that isn't important at all: "private" functions that aren't meant to be exported usually start with an underscore `_` in Python convention.  It matters more if a module _doesn't_ specify an `__all__` metadata object, but in QuTiP we always do.)

Reply:
> (a minor style note that isn't important at all: "private" functions that aren't meant to be exported usually start with an underscore _ in Python convention. It matters more if a module doesn't specify an __all__ metadata object, but in QuTiP we always do.)

Thanks, I will keep that in mind!